### PR TITLE
Finished Assignment 1

### PR DESCRIPTION
**Pull request by CodyGarciaa**
### Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
Edited the HTML a bit, but mostly the CSS to portray the frontend as intended in the Figma.

### Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

bp-blog
bp-blog

## How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'
- i only worked on files PostScreen.tsx and styles.ts

## Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

### Online sources
[//]: # 'Copy links to any tutorials or documentation that was useful to you when working on this PR'

### Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @ronniebeggs

### CODE REVIEW

File: src/screens/styles.ts [hunk 1]
```diff
@@ -5,6 +5,68 @@ export const styles = StyleSheet.create({
...
+    gap: 1,
+  },
+  post: {
+    paddingHorizontal: 15,
+    paddingTop: 8,
     backgroundColor: 'white',
   },
+  person_header: {
+    paddingTop: 15,
+    alignSelf: 'stretch',
```
Comment (ronniebeggs) [inline src/screens/styles.ts]: I don't think alignSelf is necessary.

File: src/screens/styles.ts [hunk 2]
```diff
@@ -5,6 +5,68 @@ export const styles = StyleSheet.create({
...
+    alignItems: 'center',
+    gap: 10,
+  },
+  username: {
+    fontWeight: 'bold',
+    fontSize: 13,
+  },
+  post_date: {
+    color: 'grey',
+    paddingRight: 2,
```
Comment (ronniebeggs) [inline src/screens/styles.ts:32]: Try to avoid applying padding/margins directly to text elements.

File: src/screens/styles.ts [hunk 3]
```diff
@@ -5,6 +5,68 @@ export const styles = StyleSheet.create({
...
+  da_link: {
+    color: 'blue',
+    textDecorationLine: 'underline',
+  },
+  da_image: {
+    width: '100%',
+    height: undefined,
+    aspectRatio: 1.5,
+    borderRadius: 10,
+  },
```
Comment (ronniebeggs) [inline src/screens/styles.ts]: Try to be more concise with your variable names so it's easier for your fellow devs to read/review code!

Also, we use the `camelCase` naming convention for react native CSS.

File: src/screens/PostScreen.tsx [hunk 4]
```diff
@@ -7,37 +7,65 @@ import { styles } from './styles';
...
+          causes of homelessness through a combination of shelter services, job
+          training programs, and mental health support. read more online:
+          <Text style={styles.da_link}>https://www.codetenderloin.org/</Text>
+        </Text>
+        <Image
+          style={styles.da_image}
+          source={{
+            uri: 'https://cdn.britannica.com/51/178051-050-3B786A55/San-Francisco.jpg',
+          }}
+        />
```
Comment (ronniebeggs) [inline src/screens/PostScreen.tsx]: Add `alt` text to describe the image for people with accessible screen readers.

File: src/screens/PostScreen.tsx [hunk 5]
```diff
@@ -7,37 +7,65 @@ import { styles } from './styles';
...
+        </Text>
+
+        <View style={styles.person_header}>
+          <View style={styles.name_profile}>
+            <ProfilePlaceholder />
+            <Text style={styles.username}>vppraggie</Text>
+          </View>
+          <Text style={styles.post_date}>September 21</Text>
+        </View>
+        <Text style={styles.c_text}>Thanks for sharing!</Text>
```
Comment (ronniebeggs) [inline src/screens/PostScreen.tsx]: Consider how we could use divs and flex boxes to left align the username and comment text, instead of applying a left margin to the comment text.

File: src/screens/PostScreen.tsx [hunk 6]
```diff
@@ -7,37 +7,65 @@ import { styles } from './styles';
...
+          <View style={styles.name_profile}>
+            <ProfilePlaceholder />
+            <Text style={styles.username}>daviddd</Text>
+          </View>
+          <Text style={styles.post_date}>September 20</Text>
+        </View>
+        <Text style={styles.c_text}>
+          This organization is doing amazing work tackling the complex root
+          causes of the issue.
+        </Text>
```
Comment (ronniebeggs) [inline src/screens/PostScreen.tsx]: Let's create a separate react component for the Comments to reduce rewriting code.

**Review by ronniebeggs — CHANGES_REQUESTED** [review]
Overall, looking really good so far!!! I left a few minor comments but you're on the right track. Good job!
